api/src/main/resources/META-INF/native-image/io.jsonwebtoken/jjwt-api/reflect-config.json

```diff
@@ -0,0 +1,209 @@
+[
```

Comment:
Any thoughts on generating this list at build time?
This is going to be difficult to maintain and track if/when these method signatures change.

Any ideas on creating a test harness? (Extra points if it doesn't require slowly creating a native image in order to test it).

Reply:
as for generating I have no idea either, maybe I have to make another maven-plugin for it.
for test, unfortunatly I have no idea how to make it test native fastly.